com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -1078,6 +1078,13 @@ internal void ClearStencilPartial(CommandBuffer cmd)
 
         internal void ExecuteDeferredPass(ScriptableRenderContext context, ref RenderingData renderingData)
         {
+            // Workaround for bug.
```

Comment:
Do we have this bug tracked somewhere?

Reply:
Not sure if this bug is not by design, as fixing it would require the game to stall in order to finish loading assets?

Reply:
I added FB case: https://fogbugz.unity3d.com/f/cases/1329200/